plugins/main/public/components/common/wazuh-discover/render-columns.tsx

```diff
@@ -79,6 +79,34 @@ export const wzDiscoverRenderColumns: tDataGridRenderColumn[] = [
       );
     },
   },
+  {
+    id: 'vulnerability.reference',
+    render: value => {
+      const links = (
+        <>
+          {value?.split(',').map((link, index) => (
```

Comment:
**nitpick**: the `vulnerability.reference` value seems to be an string that contains the URLs separated by `, ` (comma-whitespace). So we should transform the string to an array splitting by the same separator, `, ` (comma-whitespace)